### PR TITLE
[docs] Clarify that we only officially support the emsdk

### PR DESCRIPTION
**Pull request by kripken**
This helps with https://github.com/emscripten-core/emsdk/issues/188

Also on that page, remove some obsolete mozilla infrastructure links, and add some docs for installing specific releases and tot builds, which I noticed was missing.

### CODE REVIEW

File: site/source/docs/getting_started/downloads.rst [hunk 1]
```diff
@@ -47,6 +49,43 @@ Run the following :ref:`emsdk <emsdk>` commands to get the latest tools from Git
 
 If you change the location of the SDK (e.g. take it to another computer on an USB), re-run the ``./emsdk activate latest`` and ``source ./emsdk_env.sh`` commands.
 
+Emsdk install targets
+---------------------
+
+In the description above we asked the emsdk to install and activate ``latest``, which is the latest tagged release. That is often what you want.
+
+You can also install a specific version by specifying it, for example,
```
Comment (sbc100) [inline site/source/docs/getting_started/downloads.rst]: I think you can merge the `::` with the end of the last line of text (here and below)

File: site/source/docs/getting_started/downloads.rst [hunk 2]
```diff
@@ -6,6 +6,8 @@ Download and install
 
 .. note:: You can also :ref:`build Emscripten from source <installing-from-source>` if you prefer that to downloading binaries using the emsdk.
 
+.. note:: There are additional ways to install Emscripten than the instructions below, for example, using brew on MacOS, the package manager on your linux distro, or a Docker image, etc. However, the emsdk is the only officially supported way to use Emscripten that is supported by the Emscripten project, and the only one that we constantly test (`emsdk CI <https://github.com/emscripten-core/emsdk/blob/master/.circleci/config.yml>`_, `Emscripten GitHub CI <https://github.com/emscripten-core/emscripten/blob/master/.circleci/config.yml>`_, `Chromium CI <https://ci.chromium.org/p/emscripten-releases>`_). (While we don't officially support other ways of getting Emscripten, we definitely appreciate the efforts by third parties to `package Emscripten <https://github.com/emscripten-core/emscripten/blob/incoming/docs/process.md#packaging-emscripten>`_ for users' convenience, and we'd like to help out, please get in touch if you are such a packager!)
+
```
Comment (sbc100) [inline site/source/docs/getting_started/downloads.rst]: Can you wrap these long lines at 80?

File: site/source/docs/getting_started/downloads.rst (hunk 1 above)
Comment (kripken) [reply site/source/docs/getting_started/downloads.rst]: Not sure what you mean by "merge" here?

(This seems to be the pattern we use elsewhere, a new line, then `  ::` then a new line, then the code?)

File: site/source/docs/getting_started/downloads.rst (hunk 2 above)
Comment (kripken) [reply site/source/docs/getting_started/downloads.rst]: Sure, wrapped.

File: site/source/docs/getting_started/downloads.rst (hunk 1 above)
Comment (sbc100) [reply site/source/docs/getting_started/downloads.rst]: In restructuredtext, you can (should?) end the preceeding paragaph with a `::`.   I think it looks a little neater and is more compact, but its really nit pick

See  http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

Comment (kripken) [reply site/source/docs/getting_started/downloads.rst]: Interesting, I didn't know that.

Looks like doing it that way changes the indentation, it unindents it compared to what we have now. Not sure which is better, but we should probably be consistent everywhere?

Comment (kripken) [reply site/source/docs/getting_started/downloads.rst]: In any case that seems separate - I can open an issue for that if you think it's worth looking into?

Comment (sbc100) [reply site/source/docs/getting_started/downloads.rst]: No worries.  I agree we should be consistent.

**Pull request merged**